installer/host/make_host_osx.sh

```diff
@@ -41,8 +41,10 @@ VERSION=$(cat ${TARGET}.pro | grep VERSION | tail -n 1 | awk {'print $3'})
 sed -e s/VERVAL/${VERSION}/ -i old ${TARGET}.app/Contents/Info.plist
 # Update on server
 echo Packaging installer
-macdeployqt ${TARGET}.app -dmg
 popd
-echo ${VERSION} > latest_mac
```

Comment:
I think we still want to keep this. This let's us track the version on the server so we can prompt if a newer version of the installer is available. Although -- I haven't been updating that version much.